### PR TITLE
optimize: improving method format performance

### PR DESCRIPTION
**Pull request by yangfuhai**
### Ⅰ. Describe what this PR did

improving method format performance.

### Ⅱ. Does this pull request fix one issue?

optimize code..

### CODE REVIEW

File: spring/src/main/java/io/seata/spring/annotation/GlobalTransactionalInterceptor.java [hunk 1]
```diff
@@ -160,9 +158,17 @@ public TransactionInfo getTransactionInfo() {
...
+
+        Class<?>[] params = method.getParameterTypes();
+        int in = 0;
+        for (Class<?> clazz : params) {
+            sb.append(clazz.getName());
+            if (++in < params.length) {
+                sb.append(", ");
+            }
+        }
+        return sb.append(")").toString();
```
Comment (slievrly) [inline spring/src/main/java/io/seata/spring/annotation/GlobalTransactionalInterceptor.java]: @yangfuhai Is the performance difference from the usage of + and append?

Comment (yangfuhai) [reply spring/src/main/java/io/seata/spring/annotation/GlobalTransactionalInterceptor.java]: the performance is 10 times higher than before. this is particularly important in high concurrency.

Comment (slievrly) [reply spring/src/main/java/io/seata/spring/annotation/GlobalTransactionalInterceptor.java]: Collectors.joining is implemented via StringBuilder.append, so the difference comes from method.getName() + paramTypes and StringBuilder(method.getName()).append?

Comment (yangfuhai) [reply spring/src/main/java/io/seata/spring/annotation/GlobalTransactionalInterceptor.java]: the reason for the performance gap maybe is from Arrays.stream.map vs for-loop。

Comment (zjinlei) [reply spring/src/main/java/io/seata/spring/annotation/GlobalTransactionalInterceptor.java]: recommended to use StringJoiner

**Pull request merged**